src/ngraph_mark_for_clustering.cc

```diff
@@ -209,6 +209,8 @@ Status MarkForClustering(Graph* graph) {
       type_constraint_map["Abs"]["T"] = NGraphNumericDTypes();
       type_constraint_map["Add"]["T"] = NGraphNumericDTypes();
       type_constraint_map["AddN"]["T"] = NGraphNumericDTypes();
+      type_constraint_map["All"]["T"] = NGraphDTypes();
```

Comment:
According to the op registration, input can only be bool
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/math_ops.cc#L1219

since it is only 1 type, we can remove this line:
`type_constraint_map["All"]["T"] = NGraphDTypes();`